README.md

```diff
@@ -94,6 +94,30 @@ Or you can synchronize the entire namespace:
             [foo.core :refer :all]))
 ```
 
+##### Setting the number of threads used
+
+When multithreading is enabled, Eftest uses a single fixed-threadpool
+[`ExecutorService`][ExecutorService-Javadoc] to run all selected tests.
+
+By default, Eftest will instantiate the threadpool with the number of
+processors (cores) available to the JVM, as reported by
+[`Runtime.availableProcessors`][Runtime.availableProcessors-Javadoc].
+(NB: in some circumstances, such as [in a CircleCI test
+container][CircleCI-Config-resource_class],
+`Runtime.availableProcessors` returns an erroneous value.)
+
+[ExecutorService-Javadoc]: https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/concurrent/ExecutorService.html
+[Runtime.availableProcessors-Javadoc]: https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/Runtime.html#availableProcessors()
+[CircleCI-Config-resource_class]: https://circleci.com/docs/2.0/configuration-reference/#resource_class
+
+Users can override the default behavior by including the key `:threads`
+in the options map supplied to `run-tests` with the value being any
+positive integer:
+
+```clojure
+user=> (run-tests (find-tests "test") {:threads 4})
```

Comment:
Do you think `:thread-count` would be a less ambiguous name for the option?

Reply:
Yes, but IMHO the current name is already sufficiently unambiguous. I’ll leave this as-is unless you ask me to change it 😅

Reply:
Let's change it to `:thread-count`. I think that works better with the `:multithread?` options.